src/components/Button/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import './Button.css'
+
+function Button({ text, type, className, clickHandler, disabled }) {
+    return (
+        <button type={type} onClick={clickHandler} disabled={disabled || false} className={className}>
```

Comment:
Als je de ` || false` hebt toegevoegd voor het geval dat `disabled` undefined is, dan zou dat niet hoeven, omdat de uitkomst hetzelfde zou zijn
```suggestion
        <button type={type} onClick={clickHandler} disabled={disabled} className={className}>
```